customer-chat/README.md

```diff
@@ -0,0 +1,13 @@
+# Quick Start Tutorial - Starter Project
+
+This is a starter project you can use to as a starting point for completing the [Messenger Platform quick start tutorial](https://developers.facebook.com/docs/messenger-platform/getting-started/quick-start). 
```

Comment:
update link

Reply:
@amuramoto Done, when this review is finished I'll create the starter project and then change again.